tests/e2e/test_driver.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+	"time"
+
+	"github.com/kylelemons/godebug/pretty"
+)
+
+// TestCaseDriver knows how different TC can be executed
```

Comment:
```suggestion
// TestCaseDriver knows how different testcases can be executed
```
no need for acronym here imo